packages/react-sdk/src/state/communication/webRtcCommunicationChannel.ts

```diff
@@ -222,6 +197,67 @@ export class WebRtcCommunicationChannel implements CommunicationChannel {
     this.statisticsSubject.next(cloneDeep(this.statistics));
   }
 
+  /**
+   * If there are already known TURN servers, this is a noop.
+   * Otherwise wait for TURN servers from the Widget API until a timeout.
+   */
+  private async initTurnServers(): Promise<void> {
+    if (this.turnServers !== undefined) {
+      // TURN servers already known
+      return;
+    }
+
+    const source = race(
```

Comment:
seems like race is [being deprecated](https://rxjs.dev/api/operators/race) in rxjs v8?

Reply:
Argh - I wonder why all the projects are so fancy about breaking their APIs all the time… :neutral_face:

Reply:
Done